### ISSUE TITLE
error running with jekyll-scholar enabled

### ISSUE THREAD

**Issue by tcatapano** [minicomp/ed#11]
Ive followed the instructions on enabling jekyll-scholar, but on running `jekyll serve` I get the following error:

```
CiteProc requires the `unicode_utils` or `unicode` Gem on Ruby 2.3
jekyll 3.0.5 | Error:  undefined method `upcase' for module `CiteProc'
```
This looks like this is an issue with jekyll-scholar (actually CiteProc), see: https://github.com/inukshuk/citeproc-ruby/issues/60, but it's unclear how to fix. Any help is appreciated.

My environment:

OS Mac 10.12.6
Ruby 2.3.6
Jekyll 3.0.5
Ed ca45b087d322feae8be9045954aca20c5623fa53 (i.e., latest commit in master)

**Comment by karlstolley** [comment]
@tcatapano you're right, that's not an Ed issue. But, if you're using a Gemfile in your project, you should be able to include a line that replicates what the CiteProc people have done; here's their commit: https://github.com/inukshuk/citeproc-ruby/commit/5a76845d8779be6b6ddb4ae55cec06a5384bb793

It's worth considering, though, whether a Ruby upgrade might solve your problems and make your life better overall. Based on your Ruby at 2.3.6, I'm guessing you're using the MacOS system Ruby. Those are always several versions behind (the latest Ruby release is 2.6.1). You might look into using rbenv or RVM to install a more up-to-date Ruby. The unicode issue that that `unicode_utils` Gem fixes has apparently been addressed in Ruby 2.4

**Comment by tcatapano** [comment]
@karlstolley Thanks! Switching to Ruby 2.4 in RVM solved the problem.
